unlock-app/src/components/interface/locks/metadata/custom/AddProperty.tsx

```diff
@@ -0,0 +1,98 @@
+import { Button, Input, Modal } from '@unlock-protocol/ui'
+import { useFormContext, useFieldArray } from 'react-hook-form'
+import { RiDeleteBack2Line as DeleteIcon } from 'react-icons/ri'
+import { ComponentProps } from 'react'
+import { Attribute, MetadataFormData } from '../utils'
+
+export function AddPropertyModal({
+  isOpen,
+  setIsOpen,
+}: ComponentProps<typeof Modal>) {
+  const {
+    control,
+    register,
+    formState: { errors },
+  } = useFormContext<MetadataFormData>()
+  const {
+    fields: properties,
+    append: appendProperty,
+    remove: removeProperty,
+  } = useFieldArray({
+    control,
+    name: 'properties',
+  })
+
+  return (
+    <Modal isOpen={isOpen} setIsOpen={setIsOpen}>
+      <div className="grid gap-6">
+        <header className="space-y-2 text-brand-dark">
+          <h3 className="text-xl font-bold"> Add Properties </h3>
+          <p>
+            This custom attribute will shows up underneath of the NFT image in
```

Comment:
Let's add a link to the OpenSea docs here for clarity.

Reply:
done